lib/sqlHandler.js

```diff
@@ -209,11 +212,15 @@ SqlStore.prototype._defineRelationModel = function (relation, relationName, many
       break
   }
 
-  var relatedModel = self.sequelize.define(modelName, modelProperties, {
+  var relatedModel = self.sequelize.define(modelName, modelProperties, _.defaults({
     timestamps: false,
     indexes: [ { fields: [ 'id' ] } ],
+    underscored: self.sequelize.options.define.underscored,
+    freezeTableName: self.sequelize.options.define.freezeTableName
+  }, {
+    underscored: false,
```

Comment:
defaults on top of defaults? don't the options themselves have defaults before they get here?

Reply:
The default for `freezeTableName` is `false`. The defaults here are just restoring the original behaviour before this PR change.